spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -7223,11 +7223,10 @@ libraries), a developer could create the following TLD:
 <?xml version="1.0" encoding="UTF-8" ?>
 
 <taglib
-    xmlns="http://java.sun.com/xml/ns/j2ee"
+    xmlns="https://jakarta.ee/xml/ns/jakartaee"
```

Comment:
Should we update the namespace here before we complete https://github.com/eclipse-ee4j/jstl-api/issues/179?

Reply:
I was thinking of encouraging the use of the new namespace, but we probably should wait until 179 is completed. I'll revert the namespace changes in these examples for this PR.